docs/spelling_wordlist.txt

```diff
@@ -1,3 +1,16 @@
+CIDR
+Config
+EBS
+Elasticsearch
+Golang
+Grafana
+Istio
+Jenkins
+Logstash
+PoC
+Posibilities
+ReplicaSet
+Todo
```

Comment:
Ideally we run a `%! sort -u` on the dictionary to reduce rebases (for the next PR :slightly_smiling_face: )